Source/Core/Common/StringUtil.h

```diff
@@ -60,6 +65,17 @@ bool TryParse(const std::string& str, T* output, int base = 0)
 {
   char* end_ptr = nullptr;
 
+  if (!strcasecmp("true", str.c_str()))
+  {
+    *output = static_cast<T>(1);
+    return true;
+  }
+  else if (!strcasecmp("false", str.c_str()))
+  {
+    *output = static_cast<T>(0);
+    return true;
+  }
+
```

Comment:
ymmv but I feel this is pretty ugly to have in the middle of what is supposed to be a generic integer parsing function. At the very least I'd pass a flag that defaults to false whether true/false should convert to valid integers.

Reply:
I tried to do that, but I felt like the changes necessary were a bit invasive.